js/parser.js

```diff
@@ -1337,9 +1342,10 @@
      * https://github.com/informatics-isi-edu/ermrestjs/issues/447
      *
      * @param       {String|Object} str Can be blob or json (object).
+     * @param       {String|Object} hash to generate rediretUrl in error module.
      * @constructor
      */
-    function ParsedFacets (str) {
+    function ParsedFacets (str, hash) {
```

Comment:
Change `hash` to `path`. It's miss-leading. Hash makes sense in chaise, but not here.